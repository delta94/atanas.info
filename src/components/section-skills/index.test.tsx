import * as React from 'react';
import { shallow } from 'enzyme';

import { SectionSkills } from '.';

describe('SectionSkills component', () => {
	let useEffect: any;

	const mockUseEffect = () => {
		useEffect.mockImplementationOnce((f: () => any) => f());
	};

	beforeEach(() => {
		useEffect = jest.spyOn(React, 'useEffect');

		mockUseEffect();
	});

	it('Should render the SectionSkills component', () => {
		const wrapper = shallow(<SectionSkills />);

		expect(wrapper).toMatchSnapshot();
	});
});
