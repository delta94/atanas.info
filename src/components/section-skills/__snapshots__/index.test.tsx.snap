// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionSkills component Should render the SectionSkills component 1`] = `
<Component
  hasButton={true}
  hasShell={true}
  id="skills"
>
  <h2>
    Skills 
    <br />
    <small>
      (Drag the bubbles to play)
    </small>
  </h2>
  <div
    id="skills-graph"
  />
</Component>
`;
